# MasksAndTF
# Contains segment mask plus camera extrinsics/intrinsics 
# (everything needed for height map projection)
sensor_msgs/Image class_probs
sensor_msgs/CameraInfo cam_info